cpu/kinetis/periph/gpio.c

```diff
@@ -121,84 +123,152 @@ static inline PORT_Type *port(gpio_t pin)
 
 static inline GPIO_Type *gpio(gpio_t pin)
 {
+#if !defined(KINETIS_SERIES_E)
```

Comment:
Change the condition to depend on a new macro that you define at the top of this file, for example the `GPIO_PIDR_PID_MASK` symbol seem to only exist in the KE0x GPIO:
```c
#if defined(GPIO_PIDR_PID_MASK)
#define KINETIS_GPIO_E0 1
#endif
```
and then do all conditional compilation in this file dependent on this new `KINETIS_GPIO_E0` macro instead